src/style.css

```diff
@@ -1 +1,93 @@
-/* Styles go here */
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  margin: 0 auto;
+  --link-color-base: black;
+  --link-color-hover: rgba(0, 172, 220, 1);
+}
+
+h1 {
+  display: none;
+}
```

Comment:
it's not a good practice to use display: none for h1 tag. Either use [visually hidden snippet](https://levimcg.com/blog/visually-hidden-snippet/) or just remove h1 for this task